Projects Page restructure and New Home Banner for Impact Initiatives 2026

johnsBeharry opened the pull request:
Big refresh to the projects page for the 2026 Impact Initiatives push. New hero with animated text, updated project cards, and a fresh home page banner to match.

  What's changed

  - Projects page now opens with a bold '26 + "Impact Initiatives" header that animates in on load
  - Hero image slides in with a reveal effect
  - Each project card pulls its background color from the icon artwork
  - Added sections linking out to Community Projects and Collaborations
  - Trimmed down to 18 focused initiatives with updated descriptions
  - Funding note at the bottom pointing designers to Spiral grants
  - Home page gets a new "Creative Block" banner option
  - GSAP scripts are now local instead of loading from CDN

  For reviewers

  - Give the animations a spin on mobile and desktop
  - Skim the project descriptions for anything that reads off
  - Click through the Discord links to make sure they land correctly
  
  [PREVIEW] [bd.pxpush.com](https://bd.pxpush.com)

johnsBeharry committed:
Add projects page redesign with new components

- Add split-section component with light/dark themes and configurable backgrounds
- Add ordered-cards component for 4-card grid layout
- Add projects data file with categories and descriptions
- Add project page SVG assets and icons
- Update projects.md with new layout and dynamic content
- Add placeholder pages for collabs and community sections

johnsBeharry committed:
feat: add Creative Block header theme and update projects page styling

johnsBeharry committed:
feat: add hero animations and update banner images

johnsBeharry committed:
feat: add hero mask reveal animation and localize GSAP

johnsBeharry committed:
fix: add missing project ID 1 (Reference wallet)

johnsBeharry committed:
Update projects with correct descriptions and funding section

matbalez commented:
The changes look AMAZING! Thank you Johns!

3 suggestions:

(1) Can we lock the homepage header to your new version and disable the carousel feature? 
<img width="1447" height="825" alt="Screenshot 2025-12-20 at 4 46 38 AM" src="https://github.com/user-attachments/assets/cbee0e5c-d31d-4255-87d7-1edc92a72ac8" />

(2) On the home page, right under the fold let's insert a new subheading and brief copy that explains the Impact initiatives and offers again the CTA button to go read about them. I find it a bit odd right now that the Impact Initiatives are only mentioned ATF and nowhere else on page. My suggested copy: 

*Now's the time: let's level-up bitcoin design*
Our theme for 2026 is: Impact. Our Impact Initiatives are areas in which we believe Design can dramatically improve bitcoin, and thus, the world. It is a big ambitious slate of work, one for which we need incredible designers to lean in to help us explore, to create and to lead the charge with us. 

BUTTON: View Impact Initiatives

(3) can we make sure the hero graphic of this page (appropriately sized) is provided as the meta og tag so that when link to the /projects page is shared on twitter and other social media it will render a nice image + text blurb.

mouxdesign commented:
- Homepage: On hover confirmation: The is a finger when hovering over the buttons but there is no confirmation when a user hovers over it such a s a change in color etc.

- Projects page: The copy at the bottom reads that Spiral will offer grants. However during our discussion we mentioned that it would be great to see people interested start discussions in the BDG. That call to action to begin those discussions is not strong enough on the Projects Page IMHO.

Suggestion: Could we add something in that reads: "Start a discussion here" versus "Connect"  here.
<img width="829" height="212" alt="Screenshot 2025-12-20 at 21 26 48" src="https://github.com/user-attachments/assets/a9ce1a88-1706-445e-8072-eba580b9c82e" />

Then also add something at the closing copy. 
Closing copy suggestion:

Funding

For any designers wanting to commit to full-time design work on these impact initiatives, or similarly ambitious initiatives not yet on our radar, grants are available from [Spiral](https://spiral.xyz/)—a division of Block Inc. that promotes and funds open-source bitcoin development and adoption (and helped convene the Bitcoin Design Community 5+ years ago).

**We encourage you to kick off discussions in the Bitcoin Design Community on the project/s that you are interested in by clicking on the links above.** 

-----------

matbalez commented:
I agree we can do better than “Connect”

Maybe “Join the discussion” rather than “Start a discussion” (b/c I think
totally fine and natural for someone to join the channel to browse / lurk
for a while).

RE: the additional paragraph under the post script, I tend to think it is
unnecessary particularly if we’re using the stronger CTA described above
for each project. I think it’s fairly obvious that if someone doesn’t have
a good sense of what to do/propose/next context they will seek that context
first? And a post script of a post script seems a bit excessive.

On Sat, Dec 20, 2025 at 12:28 PM Mo ***@***.***> wrote:

> *mouxdesign* left a comment (BitcoinDesign/Guide#1206)
> <https://github.com/BitcoinDesign/Guide/pull/1206#issuecomment-3678090934>
>
>    -
>
>    Homepage: On Click confirmation: The is a finger when hovering over
>    the buttons but there is no confirmation when a user hovers over it such a
>    s a change in color etc.
>    -
>
>    Projects page: The copy at the bottom reads that Spiral will offer
>    grants. However during our discussion we mentioned that it would be great
>    to see people interested start discussions in the BDG. That call to action
>    to begin those discussions is not strong enough on the Projects Page IMHO.
>
> Suggestion: Could we add something in that reads: "Start a discussion
> here" versus "Connect" here.
> Screenshot.2025-12-20.at.21.26.48.png (view on web)
> <https://github.com/user-attachments/assets/a9ce1a88-1706-445e-8072-eba580b9c82e>
>
> Then also add something at the closing copy.
> Closing copy suggestion:
>
> Funding
>
> For any designers wanting to commit to full-time design work on these
> impact initiatives, or similarly ambitious initiatives not yet on our
> radar, grants are available from Spiral <https://spiral.xyz/>—a division
> of Block Inc. that promotes and funds open-source bitcoin development and
> adoption (and helped convene the Bitcoin Design Community 5+ years ago).
>
> *We encourage you to kick off discussions in the Bitcoin Design Community
> on the project/s that you are interested in by clicking on the links above.*
> ------------------------------
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BitcoinDesign/Guide/pull/1206#issuecomment-3678090934>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AORAHD3WFKUZGE7UWYRPKCD4CWWNJAVCNFSM6AAAAACPTYJK4OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMNZYGA4TAOJTGQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

GBKS reviewed (changes requested):
Looks awesome! Just a few small notes:

- Compress the images, you can cut them down to 25% of their current size (I use ImageOptim).
- The links in the last paragraph look just like regular text. Maybe add an underline.
- The collabs and community pages will be hard to find with this new structure, but I think that's fine for now.
- If you update the preview image (as discussed in Discord), make sure to not use the same file name of the current preview image, as services do caching at both the URL and file name level.

Nice work also using the theme variables. Did AI pick those up?

I tested on desktop Safari and Chrome, and mobile iOS Safari and Android Chrome.

swedishfrenchpress commented on projects.md:
```diff
@@ -267,76 +17,146 @@ Editor's notes
 </div>
 
-## Collaborations
+**Funding**
```
wdyt about making the funding section match the style of the 'community projects' & 'collaborations' sections? something with a graphic on the right (nice chance to use the spiral logo) and a CTA that sends the user to a spiral grant website with more info on how their grants work. the current funding section feels a bit out of place since it's the only thing italicized on the page.

johnsBeharry replied:
will revisit this after launch

swedishfrenchpress commented on _data/projects.yaml:
```diff
@@ -0,0 +1,292 @@
+    title: Branding
+    short_description: Create brand campaigns and other marketing initiatives that reframe the narrative.
+    long_description: |
+      Let's create brand campaigns and other marketing initiatives that target various constituencies within bitcoin with laser-focused messaging and compelling story-telling. Let's reframe the narrative. Let's get a bit crazy. We can lean into experimental and artistic ideas that point the way forward for the industry, and the world. Our assets and creative direction can equip and inspire builde ...
```
on my end I'm seeing merchants, savers, spenders, policymakers, designers, and population at large with two **'s instead of bold. assuming this is a markdown mistake and they should be bold? could putting the comma right after the two stars be causing this?
<img width="531" height="166" alt="image" src="https://github.com/user-attachments/assets/b5ab5729-b959-47c0-94ae-34f882e7d3b7" />

johnsBeharry replied:
fixed

johnsBeharry commented on _data/projects.yaml:
```diff
@@ -0,0 +1,292 @@
+      url: https://discord.gg/hQN5BNWH
+
+  - id: 16
+    image: project_image_01.svg
```
yes i've got to work on this since the list was consolodated

johnsBeharry replied:
they are made now to be distinct, but not happy with the iconogaphy

matbalez commented:
RE: the postscript blurb about funding being available from Spiral, I think
it is more likely to get read in its current form (as a post script) than a
separate module further down the page, so I’d say just leave as is for now?

On Sun, Dec 21, 2025 at 5:05 AM Johns Beharry ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In _data/projects.yaml
> <https://github.com/BitcoinDesign/Guide/pull/1206#discussion_r2637811431>:
>
> > +    color: "#9EE6FF"
> +    category: Design for Safety
> +    title: Multi-party self custody
> +    short_description: Design UX for multi-party self custody that maximizes security while minimizing fraud as well as considering transparency and the transfer of power.
> +    long_description: |
> +      Let's design simple UX and workflows for multi-party self custody. We can explore examples ranging from small non-profits to small businesses to strategic reserves for nation states. The UX will need to consider maximizing coin security, maximizing personal safety, minimizing fraud as well as considering transparency and the transfer of power. Important work!
> +    key_people:
> +      - handle: "Maybe you?"
> +        url: null
> +    status: Planning stage
> +    connect:
> +      channel: "#general"
> +      url: https://discord.gg/hQN5BNWH
> +
> +  - id: 16
> +    image: project_image_01.svg
>
> yes i've got to work on this since the list was consolodated
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BitcoinDesign/Guide/pull/1206#discussion_r2637811431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AORAHD25FIVGZ57MJMKC3V34C2LI3AVCNFSM6AAAAACPTYJK4OVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTMMBRGY4TMNZVG4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

johnsBeharry committed:
feat(home): add Impact Initiatives callout and lock Creative Block theme

johnsBeharry committed:
chore(images): optimize PNG assets and add OG images for projects page

johnsBeharry committed:
style(header): flip text to be above image on mobile

johnsBeharry committed:
feat(projects): improve OG tags and add nav_title support

johnsBeharry committed:
chore(projects): change text from connect to join...

matbalez commented:
thanks for the changes @johnsBeharry LGTM

one small nit:

on the home page the new subheading (Now’s the time: let’s level-up bitcoin design) looks to be smaller font size than the one below it (We are helping make bitcoin more intuitive and accessible.)

otherwise, let's SHIP THIS THING!

mouxdesign commented:
<img width="1020" height="1016" alt="mogashni_playful_colorful_illustration_of_a_boat_with_ballons_o_48623221-2d87-423f-a6e7-0b9ba936846a 1" src="https://github.com/user-attachments/assets/7472e476-bc94-4d39-a5fe-3e7637c6f428" />

johnsBeharry committed:
fix(projects): render markdown in projects list description reported by @swedishfrenchpress

johnsBeharry committed:
style: improve projects page styling and fix content issues

- Fix markdown bold formatting in projects YAML by moving commas inside markers
- Enable markdown processing in project descriptions with markdownify filter
- Fix bold text color inheritance with !important to override global styles
- Add underlines to footer links for better visibility
- Remove spacing between split sections on mobile
- Update split section background graphics with new designs
- Fix duplicate project images for projects 16 and 17
- Increase Impact Initiatives heading size to h2 for consistency

johnsBeharry committed:
style(mobile): improve spacing and typography on projects page

- Add spacing after projects grid on mobile
- Increase intro text size to 24px on mobile
- Increase intro text line-height to 1.4 on mobile for better readability

mouxdesign commented:
To add in links to two Discord channels

- Edit copy here to Humans of Bitcoin
- Add link to Discord channel https://discord.gg/FGvTWr3D

<img width="868" height="556" alt="Screenshot 2025-12-22 at 13 22 58" src="https://github.com/user-attachments/assets/54b9f953-fe0f-490e-bbd9-31ce03f5e6ad" />

- Add in link to join BIP 353 Discord channel here

<img width="872" height="634" alt="Screenshot 2025-12-22 at 13 23 06" src="https://github.com/user-attachments/assets/6f6e37c6-f4a6-4636-83ea-1e30545f92c2" />

- Link for channel: https://discord.gg/M7nJksmJ

johnsBeharry committed:
feat(projects): add Discord channel links for Humans of Bitcoin and BIP-353

- Add dedicated Discord channel link for Humans of Bitcoin project
- Update channel name from #general to #humans-of-bitcoin
- Add Discord channel link for BIP-353 (Human Bitcoin Addresses) project

johnsBeharry commented on _data/projects.yaml:
```diff
@@ -0,0 +1,292 @@
+      url: https://discord.gg/hQN5BNWH
+
+  - id: 17
+    image: project_image_02.svg
```
fixed

johnsBeharry committed:
optimize: compress projects page images

Optimized 30 SVG and PNG files used on /projects page. Reference @GBKS for compression work.

johnsBeharry reviewed (commented):
**@matbalez**
  * Locked the homepage header to the Creative Block version. Carousel is fully disabled now — only the Creative Block header shows.
  * Added the under-the-fold Impact Initiatives explainer with CTA. Also fixed the type hierarchy: the new subheading is now an `h2`, consistent with the rest of the homepage sections.
  * OG/meta handling for `/projects` is implemented (custom title + description for SEO/social instead of the generic “Projects”). I’m doing one final sanity check, but structurally this is in place.
  * Replaced **“Connect” → “Join the discussion”** everywhere.
  * Kept the funding note as a postscript for now. I had a vague memory of a weekend comment about possibly referencing BDF funding as well, but can’t locate it anymore — so nothing changed there for this pass.

**@mouxdesign**

  * Header theme tiles on home currently don’t have hover states — that applies to all theme headers, not just Creative Block. I didn’t sneak in a partial fix here; happy to roll hover/active states for *all* theme headers into a follow-up update.
  * Updated both Discord links as requested: **Humans of Bitcoin** and **BIP-353** are now live and correctly wired.
  * All newly added links are now underlined, so they should read clearly as links.

**@GBKS**

  * All images related to projects compressed using ImageOptim, thanks for the nudge.
  * Final-paragraph links are now visually distinct (underlined), same change as above.
  * Preview / OG images were updated with new filenames to avoid caching issues.
  * No regressions from theme variable usage — haha yes, the AI bots were wielded with care.

**@swedishfrenchpress**

  * Funding section styling is unchanged for now. Given timing + copy considerations, I left it as-is for launch, but I agree it could be stronger and more integrated — happy to revisit with a more intentional CTA in a later update.
  * Markdown rendering bug for bold text (merchants, savers, policymakers, etc.) is fixed — thanks for catching that one.
  * Funding remaining in postscript/italic form wasn’t a super hard philosophical decision so much as a pragmatic “ship it cleanly” call; open to reshaping this once we loop back on funding CTAs more holistically.

If I missed anything blocking merge, shout — otherwise this should be in a good place to ship.

johnsBeharry committed:
optimize: replace supergraphic SVG with smaller PNG

Converted supergraphic--community-projects-grid from SVG to PNG for reduced file size.

GBKS reviewed (approved):
ACK.
